onesignalclient/notification.py

```diff
@@ -43,6 +46,31 @@ def include_player_ids(self, value):
 
         self._include_player_ids = value
 
+    @property
+    def include_segments(self):
+        return self._include_segments
+
+    # notification.include_segments = [Notification.SEGMENT_ALL]    
+    @include_segments.setter
+    def include_segments(self, value):
+        if self.mode != self.SEGMENTS_MODE:
+            raise TypeError('Mode should be set to segment.')
+
+        if not isinstance(value, list):
+            raise TypeError('Value must be a list.')
+        
+        if len(value) > 3:
+            raise TypeError('Invalid segment count! Segments available: SEGMENT_ALL, SEGMENT_ACTIVE_USERS, SEGMENT_INACTIVE_USERS')
+        else:
+            for item in value:
+                if item not isinstance(value, str):
```

Comment:
There is a syntax error here, we can fix it by changing this condition to `if not isinstance(item, str):` :)